src/day02_shift_array_insert/ArrayInsertShift.java

```diff
@@ -0,0 +1,18 @@
+package day02_shift_array_insert;
+
+public class ArrayInsertShift {
```

Comment:
Watch your naming.  Instructions state: "Write a function called `insertShiftArray` which takes in an array and the value to be added. Without utilizing any of the built-in methods available to your language, return an array with the new value added at the middle index."